test/jdk/com/sun/crypto/provider/Cipher/AEAD/GCMParameterSpecTest.java

```diff
@@ -36,7 +42,7 @@
  */
 public class GCMParameterSpecTest {
 
-    private static final int[] IV_LENGTHS = { 96, 8, 1024 };
+    private static final int[] IV_LENGTHS = { 96, 8 };
```

Comment:
Please explain the removal of the tests using IV length 1024.

Reply:
see https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/499#issuecomment-1098595922 for reason why IV > 128Bit is failing w/ OpenSSL3 and https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/499#issuecomment-1099097885 for request of change to the failing unit test

Reply:
We know why, Keith is asking for a comment in the code to explain it, so it's easier for future viewers to understand why it changed.

Reply:
I didn't know why, and yes, the place for the answer is in a comment in the code.

Reply:
Edited

Reply:
@keithc-ca are you happy with the added comment?